docs/oidc.rst

```diff
@@ -147,6 +149,23 @@ scopes in your ``settings.py``::
     If you want to enable ``RS256`` at a later date, you can do so - just add
     the private key as described above.
 
+
+RP-Initiated Logout
+~~~~~~~~~~~~~~~~~~~
+This feature has to be enabled separately as it is an extension to the core standard.
+
+.. code-block:: python
+
+   OAUTH2_PROVIDER = {
+       # OIDC has to be enabled to use RP-Initiated Logout
+       "OIDC_ENABLED": True,
+       # Enable and configure RP-Initiated Logout
+       "OIDC_RP_INITIATED_LOGOUT": True,
```

Comment:
Should be "OIDC_RP_INITIATED_LOGOUT_ENABLED"

Reply:
Good catch, Thanks!